Orange/widgets/model/owadaboost.py

```diff
@@ -0,0 +1,116 @@
+from AnyQt.QtCore import Qt
+
+from Orange.base import Learner
+from Orange.data import Table
+from Orange.ensembles import SklAdaBoostLearner
+from Orange.modelling import SklTreeLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+from Orange.widgets.widget import Msg
+
+
+class OWAdaBoost(OWBaseLearner):
+    name = "AdaBoost"
+    description = "An ensemble meta-algorithm that combines weak learners " \
+                  "and adapts to the 'hardness' of each training sample. "
+    icon = "icons/AdaBoost.svg"
+    priority = 80
+
+    LEARNER = SklAdaBoostLearner
+
+    inputs = [("Learner", Learner, "set_base_learner")]
+
+    #: Algorithms for classification problems
+    algorithms = ["SAMME", "SAMME.R"]
```

Comment:
According to http://scikit-learn.org/stable/auto_examples/ensemble/plot_adaboost_multiclass.html, SAMME.R is better. It is also the default.